core/src/main/java/org/kohsuke/stapler/HttpResponses.java

```diff
@@ -226,6 +243,21 @@ public void generateResponse(StaplerRequest req, StaplerResponse rsp, Object nod
         };
     }
 
+    /**
+     * Serves a plain text response.
+     */
+    public static HttpResponse text(final String text) {
```

Comment:
`@since` is basically useless for Jenkins developers. Will add a changelog entry when merging.